docs/install/debian-ubuntu.rst

```diff
@@ -23,29 +23,22 @@ the CrateDB package repository::
     sudo apt install --yes apt-transport-https apt-utils curl gnupg lsb-release
 
     # Import the public GPG key for verifying the package signatures.
-    curl -sS https://cdn.crate.io/downloads/deb/DEB-GPG-KEY-crate | sudo tee /etc/apt/trusted.gpg.d/cratedb.asc
+    curl -sS https://cdn.crate.io/downloads/apt/DEB-GPG-KEY-crate | \
+        sudo tee /etc/apt/trusted.gpg.d/cratedb.asc
```

Comment:
Maybe we can also have the key at https://cdn.crate.io/downloads/debian/DEB-GPG-KEY-crate?

Reply:
The situation on the server machine is like that....
![image](https://github.com/crate/cratedb-guide/assets/453543/57a6aa73-a4f0-424d-96ea-afee7d4658e6)

... and I haven't been able to find the root source of the `DEB-GPG-KEY-crate` file within any of our repositories. In this spirit, because I am unsure where to target the symlink to, and in order to keep things self-contained, I will copy that file to the `debian` folder as well.

Reply:
Done.

![image](https://github.com/crate/cratedb-guide/assets/453543/bdd919dd-1e64-4a99-988a-9b887c5f28c6)